students/JerryH/Lesson03/mailroom.py

```diff
@@ -1,79 +1,93 @@
 #!/usr/bin/env python3
 
-#List of sample donors and donations
+# List of sample donors and donations
 donors = [
-['Bill Gates', [234.22, 45645.24, 43953.09, 98823]],
-['Jeff Bezo', [4564.23]],
-['Mike Dell', [299.09, 26273.67]],
-['Harry Potter', [8234.09, 83948.04, 7834.23]],
-['Ben Williams', [29283.00, 1334.34]],
-['Guy James', [93.00, 34.34]]
-]
+    ['Bill Gates', [234.22, 45645.24, 43953.09, 98823]],
+    ['Jeff Bezo', [4564.23]],
+    ['Mike Dell', [299.09, 26273.67]],
+    ['Harry Potter', [8234.09, 83948.04, 7834.23]],
+    ['Ben Williams', [29283.00, 1334.34]],
+    ['Guy James', [93.00, 34.34]]
+    ]
 
 THANK_YOU_OPT, REPORT_OPT, QUIT_OPT = (1, 2, 3)
 
+
 def prompt():
-    return int(input("Please choose the following options:\n1) Send a Thank you.\n2) Create a Report.\n3) Quit.\n"))
+    return int(input("Please choose the following options:"
+        "\n1) Send a Thank You.\n2) Create a Report.\n3) Quit.\n"))
+
+
+def donor_name_prompt():
+    return input("Send a Thank You - Please enter a full name or type \"list\""
+        "to list the current donors:\n")
 
-def thank_you_prompt():
-    return input("Send a Thank You - Please enter a full name or type \"list\" to list the current donors:\n")
 
 def get_all_donor_names(donor_list):
     all_names = []
     for i in range(len(donor_list)):
         all_names.append(donor_list[i][0])
     return all_names
 
-def main( op = 0 ):
-    while op > 3 or op < 1:
-        op = prompt()
 
-        if op == THANK_YOU_OPT: #Append or list donors
-            #Add donor's name or list the exisiting donors
-            d_name = thank_you_prompt()
-            while d_name.lower() == "list": #provide a list of donors if "list" is entered
-                #Get each unique donor's name from the list
-                for each_name in sorted(get_all_donor_names(donors)):
-                    print("{}".format(each_name))
-                d_name = thank_you_prompt()
+def list_all_donor_names(donor_list):
+    for each_name in sorted(get_all_donor_names(donor_list)):
+        print("{}".format(each_name))
```

Comment:
from what I see here neither `list_all_donor_names` or `get_all_donor_names` need the `donor_list` param, since it's just referring to the global `donors` var